tokenizers/src/normalizers/strip.rs

```diff
@@ -36,3 +37,91 @@ impl Normalizer for Strip {
         Ok(())
     }
 }
+
+/// This normalizer removes combining marks from a normalized string
+/// It's different from unidecode as it does not attempt to modify
+/// non ascii languages.
+#[derive(Copy, Clone, Debug)]
+pub struct StripAccents;
+impl_serde_unit_struct!(StripAccentsVisitor, StripAccents);
+
+impl Normalizer for StripAccents {
+    /// Strip the normalized string inplace
+    fn normalize(&self, normalized: &mut NormalizedString) -> Result<()> {
+        let mut transforms = Vec::with_capacity(normalized.get().len());
```

Comment:
You can use `NormalizedString::filter` to avoid having to do this manually:
```rust
normalized.filter(|c| !is_combining_mark(c))
```

Reply:
That actually fixes a bug. :D